azerothcore/azerothcore-wotlk
# fix(DB/Gossip): Arcanist Ithanas gossip

## Changes Proposed:
-  Added new gossip for Arcanist Ithanas cause it was not in npc_text
-  Created new gossip menu cause Arcanist doesn't have one
-  Added condition for the second gossip cause it must be shown only when ```A Fistful of Slivers``` quest completed

## Issues Addressed:
- Closes #6734 
- Closes chromiecraft/chromiecraft#1097

## SOURCE:
- https://wowpedia.fandom.com/wiki/Arcanist_Ithanas

## Tests Performed:
- Tested on local server

## How to Test the Changes:
1.  Go to ```Arcanist Ithanas``` and talk with him. You will see the first gossip
2. Take and complete ```A Fistful of Slivers```
3. Return to ```Arcanist Ithanas``` and talk with him. You will see the second gossip



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
